### DISCUSSION TITLE
Last contribution date - how is this determined?

### DISCUSSION THREAD

**Discussion by michaelmoss** [community/OSPO#59] [General]
In the graphql api, it's easiest to retrieve this by using the last contribute date on the default branch. I'm curious how this date is determined. If it's based just on commits, and if so, if its latest across all branches. Thank you!

**Comment by ahpook** [comment]
"Contributions" in the dashboard include Issues, PRs, comments, and commits. The computation is not immediate though, it's run periodically so (if I understand the backend stuff correctly) it'll not show anything newer than "yesterday".

**Comment by devmuhib009** [comment]
The last contribution date in GitHub Community is determined by various activities, including commits, pull requests, issues, and comments across repositories, but may have a slight delay in updating, typically showing activity up to "yesterday."
